webapp/javascript/pages/ExploreView.tsx

```diff
@@ -0,0 +1,304 @@
+import React, { useEffect } from 'react';
+import type { Maybe } from 'true-myth';
+import type { ClickEvent } from '@szhsin/react-menu';
+import Color from 'color';
+
+import type { Profile } from '@pyroscope/models/src';
+import Box from '@webapp/ui/Box';
+import Toolbar from '@webapp/components/Toolbar';
+import TimelineChartWrapper, {
+  TimelineGroupData,
+} from '@webapp/components/TimelineChartWrapper';
+import { FlamegraphRenderer } from '@pyroscope/flamegraph/src/FlamegraphRenderer';
+import Dropdown, { MenuItem } from '@webapp/ui/Dropdown';
+import useColorMode from '@webapp/hooks/colorMode.hook';
+import useTimeZone from '@webapp/hooks/timeZone.hook';
+import { useAppDispatch, useAppSelector } from '@webapp/redux/hooks';
+import {
+  actions,
+  setDateRange,
+  fetchTags,
+  selectQueries,
+  selectContinuousState,
+  selectAppTags,
+  TagsState,
+  fetchExploreView,
+  fetchExploreViewProfile,
+} from '@webapp/redux/reducers/continuous';
+import { queryToAppName } from '@webapp/models/query';
+
+import styles from './ExploreView.module.scss';
+
+const timelineSeriesColors = [
```

Comment:
lets try using this array [here](https://github.com/pyroscope-io/pyroscope/blob/main/packages/pyroscope-flamegraph/src/FlameGraph/FlameGraphComponent/colorPalette.ts#L22) (ideally we could import it somehow?)